### DISCUSSION TITLE
Bring local branch up to current v2_devel

### DISCUSSION THREAD

**Discussion by microtechno9000** [automatic-ripping-machine/automatic-ripping-machine#508] [Q&A] [answered]
Hi All

Started playing around with some of the ARM code, and created a branch os_vals.

I see that the main arm v2_devel has made some changes recently that I would like to merge into my branch prior to creating a pull request. 

Anyone know how to do that?

Edit: ok, so realised that I had my server running the 1337-server version of ARM, and laptop on the main ARM.

**Comment by 1337-server** [comment, marked as answer]
Click on the 'code' tab on your fork.

then look for this -> https://imgur.com/a/c604o7m

Click the fetch upstream, and it should open a pull request to update your fork with the recent changes
